app/models/user.rb

```diff
@@ -32,6 +32,10 @@ def superadmin_or_department_manager?
     superadmin? || department_manager?
   end
 
+  def department?
```

Comment:
There is the method `manager_with_department?`, but i'm thinking, that this method should be named that way and the other `department?`, your opinion?